controllers/virtualmachine/virtualmachinegroup/virtualmachinegroup_controller.go

```diff
@@ -182,23 +185,22 @@ func (r *Reconciler) ReconcileNormal(ctx *pkgctx.VirtualMachineGroupContext) (ct
 		}
 	}(ctx.VMGroup.Status.DeepCopy())
 
-	if err := r.reconcileMembers(ctx); err != nil {
+	var err error
```

Comment:
Ditch the line `var err error` and instead make the function return named variables, such as `(_ ctrl.Result, reterr error)`. That way there's no problem distinguishing between locally scoped `err` variables in the function and the one returned.

Reply:
Updated.